instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/SpanSuppressors.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.internal.SpanKey;
+import java.util.Map;
+import java.util.Set;
+
+final class SpanSuppressors {
```

Comment:
just curious if this change is related to the jApiCmp checks somehow? thx (no objection to the change itself)

Reply:
Yep -- all these implementations were public by default, since they were defined inside an interface (even if the interface itself was package-private). I moved them to a package private class to make sure they're not accidentally leaked out as public API

Reply:
oh! 👍